js/ad_manager_template.js

```diff
@@ -35,20 +33,15 @@ OO.Ads.manager(() => {
      * @param {object} adManagerController A reference to the Ad Manager Controller
      * @param {string} playerId The unique player identifier of the player initializing the class
      */
-    this.initialize = (adManagerController, playerId) => {
+    this.initialize = (adManagerController) => {
```

Comment:
Not sure about this file. This is a boilerplate for the ad manager and it should be like an example of how to implement it. Maybe it's better to revert and disable some rules for this file. @ooyala/player-web ?